.github/workflows/kind-e2e.yaml

```diff
@@ -65,25 +65,25 @@ jobs:
       fail-fast: false # Keep running if one leg fails.
       matrix:
         k8s-version:
-        - v1.31.0
-        - v1.30.4
-        - v1.29.8
+        - v1.34.1
+        - v1.33.5
+        - v1.32.9
         mode:
         - ha
         - non-ha
         # Map between K8s and KinD versions.
         # This is attempting to make it a bit clearer what's being tested.
         # See: https://github.com/kubernetes-sigs/kind/releases/tag/v0.11.1
         include:
-        - k8s-version: v1.31.0
+        - k8s-version: v1.34.1
           kind-version: v0.24.0
```

Comment:
The kind version referenced does not provide prebuilt images for the specified versions and the kind image checksum and version references are mismatching